docs/releases/2.5.1.rst

```diff
@@ -0,0 +1,74 @@
+Translate Toolkit 2.5.1
+***********************
+
+*Released on 25 April 2020*
+
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- PO
+
+    - Avoid adding extra space on empty comment lines
+    - Several performance improvements
+
+- Android resources
+
+    - Workaround broken plural handling for languages missing other tag
+    - Fix setting rich content with comments
+
+- YAML
+
+    - Fixed handling of empty keys
+    - Improved round trip preserving of comments and style
+
+- TS
+
+    - Avoid duplicating plurals definition
+    - Fixed possible crash on adding new translations
+
+- INI
+
+    - Now supported on Python 3 thanks to iniparse support for it
+
+- Catkeys
```

Comment:
We usually sort the formats by their importance. Catkeys is certainly not of widespread usage.